src/content/app/tools/blast/views/blast-submission-results/BlastSubmissionResults.tsx

```diff
@@ -40,6 +38,19 @@ import type { BlastResult } from 'src/content/app/tools/blast/state/blast-result
 import type { Species } from 'src/content/app/tools/blast/state/blast-form/blastFormSlice';
 import type { BlastJob } from 'src/content/app/tools/blast/types/blastJob';
 
+import ShowHide from 'src/shared/components/show-hide/ShowHide';
+import {
+  DataTableState,
+  TableAction,
+  type TableCellRendererParams,
+  type DataTableColumns,
+  type TableData
+} from 'src/shared/components/data-table/dataTableTypes';
+import DataTable from 'src/shared/components/data-table/DataTable';
+import BlastSequenceAlignment from '../../components/blast-sequence-alignment/BlastSequenceAlignment';
+import { BlastSequenceAlignmentInput } from '../../components/blast-sequence-alignment/blastSequenceAlignmentTypes';
+import { DatabaseType } from '../../types/blastSettings';
```

Comment:
Despite resolved Imran's comment, the imports haven't been changed to type imports.